airbyte-analytics/src/main/java/io/airbyte/analytics/SegmentTrackingClient.java

```diff
@@ -95,6 +95,9 @@ public void track(String action, Map<String, Object> metadata) {
     final Map<String, Object> mapCopy = new HashMap<>(metadata);
     final TrackingIdentity trackingIdentity = identitySupplier.get();
     mapCopy.put(AIRBYTE_VERSION_KEY, trackingIdentity.getAirbyteVersion());
+    if (metadata.containsKey(USER_EMAIL_KEY_PLACEHOLDER) && metadata.get(USER_EMAIL_KEY_PLACEHOLDER).equals(USER_EMAIL_VALUE_PLACEHOLDER)) {
```

Comment:
I put this little placeholder trick so we can tweak if we want to inject user email or not...

If we are okay with always sending user email for both "Connector Jobs" and "New Connection - Backend", then this is not needed...

(as commented here: https://github.com/airbytehq/airbyte/issues/2163#issuecomment-789092781)

Reply:
@cgardens just want to make sure we are not storing the email if someone opted out.

Reply:
I am in favor of option2 let's keeps things simple, we can always revisit if needed

Reply:
I'm ok with option 2. are you okay with it @cgardens

Reply:
yeah. option 2 sounds good.